### PR TITLE
Boilerplate code + package name

### PR DESCRIPTION
**Pull request by ledor473**
- Removed the boilerplate code in the multiple operations class
- Fixed the packages names
- Removed Guava dependencies

### CODE REVIEW

File: discovery/src/main/java/com/ticketmaster/api/discovery/operation/Operation.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  public T withParam(String key, String value) {
+    return withParam(key, value, false);
+  }
+
+  public T withParam(String key, String value, boolean emptyAllowed) {
+    Preconditions.checkNotNull(value);
+    if (!emptyAllowed) {
+      Preconditions.checkArgument(!value.trim().isEmpty());
+    }
+
```
Comment (eric-karl) [inline discovery/src/main/java/com/ticketmaster/api/discovery/operation/Operation.java:42]: Could this be made private and have a public method "withOptionalParam" or similar name public instead? I don't mind boolean flags for simple logic like this but I believe they at least should be hidden.

**Comment by eric-karl** [comment]
:+1:

File: discovery/src/main/java/com/ticketmaster/api/discovery/operation/Operation.java (hunk 1 above)
Comment (ledor473) [inline discovery/src/main/java/com/ticketmaster/api/discovery/operation/Operation.java:42]: I'll make it private for now.  If we find a situation that a blank string should be allowed, we'll find a correct name for it

Comment (eric-karl) [inline discovery/src/main/java/com/ticketmaster/api/discovery/operation/Operation.java:42]: Works for me. :)

**Pull request merged**